.github/workflows/clone-db.yaml

```diff
@@ -1,4 +1,4 @@
-name: Clone Staging Database
+name: Clone Stable Database
```

Comment:
```suggestion
# This workflow runs at the top of every hour and can be manually run as needed
# The workflow will copy the database from stable (production) to our staging sandbox.

name: Clone Stable Database
```

Reply:
Also if the database is missing columns/tables in staging that exist in stable then we do expect this workflow to fail temporarily, correct? If so update this comment to reflect that as that is a very real possibility

Reply:
Yes we do, we will need to ensure that the target database has a compatible schema with the source database.